drivers/hmis/app/graphql/mutations/ce/create_ce_referral.rb

```diff
@@ -11,16 +11,15 @@ class Ce::CreateCeReferral < CleanBaseMutation
     argument :opportunity_id, ID, required: true
     argument :client_id, ID, required: true
     field :referral, Types::HmisSchema::CeReferral, null: false
-    argument :input, Types::HmisSchema::CeReferralInput, required: true
+    argument :input, Types::HmisSchema::CeReferralInput, required: false, deprecation_reason: 'Use AssignReferralParticipants mutation instead'
```

Comment:
This is going to be a breaking change since they keyword args is removed below. I think we might as well go ahead and just remove the argument entirely, since breaking changes are ok at this stage. That will speed things up so we don't have to come back and adjust this later.